migrations/migration-9/index.js

```diff
@@ -112,16 +116,18 @@ async function pinsToDAG (blockstore, datastore, pinstore, onProgress) {
   }
 
   onProgress(100, 'Updating pin root')
-  const pinRoot = new dagpb.DAGNode(new Uint8Array(), [
-    await pinset.storeSet(blockstore, PinTypes.recursive, recursivePins),
-    await pinset.storeSet(blockstore, PinTypes.direct, directPins)
-  ])
-  const buf = pinRoot.serialize()
-  const cid = await dagpb.util.cid(buf, {
-    cidVersion: 0
-  })
+  const pinRoot = {
+    Links: [
+      await pinset.storeSet(blockstore, PinTypes.direct, directPins),
```

Comment:
what is `storeSet` returning? because `dagPb.encode()` is going to be strict about them needing to be `PBLink`

are we also back at the `Tsize` issue, do we need to revisit https://github.com/ipld/js-dag-pb/issues/5?

and don't forget `dagPb.prepare()` is available to get well-formed objects pre-encode, I'm not sure that's helpful or not

Reply:
The return is [here](https://github.com/ipfs/js-ipfs-repo-migrations/pull/98/files#diff-13302036f214d8fe24b856c3306461424f4df97827050bfdc5c2fb717ee05234R269-R273), but it's:

```js
{ Name: string, Tsize: number, Hash: CID }
```